### ISSUE TITLE
My shortcuts are not working. 

### ISSUE THREAD

**Issue by TheEA66** [LibreSprite/LibreSprite#70]
Shortcuts like ctrl z and others don't seem to work. I tried changing languages and that still didn't work.

**Comment by PerryHugh** [comment]
Strange, I don't have Windows 10 so I can't confirm. Can you give some more information, what precise action are you trying to do and on what screen?

**Comment by TheEA66** [comment]
I don't really know why it is happening. I already somehow fixed that issue but it came up again. I can click on tools and manually click on undo button, but if I try to press B (Brush) it doesn't switch to that tool. Same with other tools and undo/redo buttons

**Comment by TheEA66** [comment]
I think I found out what might cause this. Whenever I use the snip and sketch on the program the shortcuts stop working.

**Comment by PerryHugh** [comment]
Strange, I can't seem to replicate it on Linux. If you are willing, a screencap uploaded to [Streamable](https://streamable.com) would be awesome, it would help narrow down the actions leading up to shortcuts failing.

**Comment by TanjaPrini** [comment]
Hi,
Are there any updates on this issue? I never had this happen to me before until today. The only way to get the shortcuts to work again is to restart libresprite, and even then they stop working after a while. I am trying to find a specific cause to this but it just seems to happen randomly.

**Comment by MintStudios** [comment]
What OS are you on?

**Comment by TanjaPrini** [comment]
I'm on Windows 10.

**Comment by leveljaap** [comment]
Hi sorry if this is a necro but I have the same issue. I'm on windows 10 and shortcuts just don't work at all. They don't do anything. Can't use ctrl + z etc or any of the tools. Clicking them manually works fine. I've never had shortcuts work in the first place though, it happens all the time when I use the program.

**Comment by jjhaggar** [comment]
I am on Windows 10. and yeah, I can confirm that this happens **sometimes**. I still haven't found what could be the cause. Normaly all the shortcuts work just fine, but sometimes they stop working (all of the shortcuts at the same time), and I'm afraid I have no idea of why this happens ^^U

Previously it happened to me when I was working both with LibreSprite and Aseprite at the same time, so I thought it could be some kind of incompatibility, but I have them both opened right now and their respective shortcuts work just fine :S

**Comment by chungchunwang** [comment]
Hi! For anyone with this issue, on Windows 11 I can get LibreSprite to recognize shortcuts by running it as administrator.

**Comment by felipemanga** [comment]
Does this still happen on version 1.1?

**Comment by chungchunwang** [comment]
Yep, I tried it with both 1.0 and 1.1. In both versions shortcuts are only recognized when running as administrator.
